docs/src/languages/java.md

```diff
@@ -1,6 +1,170 @@
 # Java
 
-Java language support in Zed is provided the [zed Java extension](https://github.com/zed-extensions/java).
-Report issues to: [https://github.com/zed-extensions/java/issues](https://github.com/zed-extensions/java/issues)
+Java language support in Zed is provided the [zed Java extension](https://github.com/zed-extensions/java) and [zed-java-eclipse-jdtls](https://github.com/ABckh/zed-java-eclipse-jdtls). You may choose one between them to satisfy your needs.
+
+## Java extension for Zed
 
 - Tree Sitter: [tree-sitter/tree-sitter-java](https://github.com/tree-sitter/tree-sitter-java)
+- Language Server: [eclipse-jdtls/eclipse.jdt.ls](https://github.com/eclipse-jdtls/eclipse.jdt.ls)
```

Comment:
Sorry, but I just noticed that this line is different from line 126 even though both extensions use the same language server.